mock-mhs-adaptor/src/main/java/uk/nhs/adaptors/mockmhsservice/producer/InboundProducer.java

```diff
@@ -0,0 +1,19 @@
+package uk.nhs.adaptors.mockmhsservice.producer;
+
+import org.springframework.jms.JmsException;
+import org.springframework.jms.core.JmsTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RestController;
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class InboundProducer {
+
+    private final JmsTemplate jmsTemplate;
+    private final String queueName = System.getenv().getOrDefault("GP2GP_MHS_INBOUND_QUEUE", "inbound");
```

Comment:
Take a look at TaskDispatcher how we get variable

Reply:
Also make sure this is in the Dockerfile so it can be set